wolfcrypt/src/signature.c

```diff
@@ -242,8 +245,7 @@ int wc_SignatureVerifyHash(
             }
     #endif /* WOLFSSL_CRYPTOCELL */
             if (ret != 0) {
-                WOLFSSL_MSG("RSA Signature Verify difference!");
-                ret = SIG_VERIFY_E;
+                WOLFSSL_MSG("RSA Signature Verify failed!");
```

Comment:
Noting that this no longer explicitly sets the error value but propagates the underlying crypto error...should be fine but worth noting in case it causes troubles.